tools/stress-cluster/cluster/azure/main.bicep

```diff
@@ -152,6 +159,7 @@ output SECRET_PROVIDER_CLIENT_ID string = cluster.outputs.secretProviderClientId
 output CLUSTER_NAME string = cluster.outputs.clusterName
 output CONTAINER_REGISTRY_NAME string = containerRegistry.outputs.containerRegistryName
 output APPINSIGHTS_KEY_SECRET_NAME string = appInsightsInstrumentationKeySecretName
+output APPINSIGHTS_CONNECTION_STRING_SECRET_NAME string = appInsightsConnectionStringSecretName
```

Comment:
Because connection string contains semicolon, when source ${ENV_FILE}, the value will be truncated.  I use double quote to escape the semicolon and it works for me.  
So here we can change to `'"${appInsightsConnectionStringSecretName}"'`, and the .env file will be `APPINSIGHTS_CONNECTION_STRING="*******"` (with double quote). 
But I am not sure this change will have any impact for other languages since they are using libraries to load .env file.

Reply:
I'm not sure where @benbp landed on this but we were considering just making two separate files, rather than trying to have one conform to both.

So one file would be a sourcable .sh file (which is nice if you exec -it into a container and want to load up the environment) and a normal .env file.

Reply:
Yeah it's going to take a little bit more plumbing to process the outputs from the new-testresources.ps1 deployment script, but should be possible. I was planning on heading in that direction but this has dropped down the priority list a little bit.